src/pyHepGrid/src/runSlurmjob.py

```diff
@@ -79,6 +79,8 @@ def _get_production_args(self, runcard, tag, baseSeed, producRun, threads,
         else:
             args["stdoutfile"]=self.get_stdout_dir_name(args["runcard_dir"])+"slurm-%j.out"
         args = self.__do_common_args(args, threads, queue)
+        # Add arguments coming from the parent interface
+        args = super().include_arguments(args)
```

Comment:
Do I understand it correctly that this allows setting arbitrary arguments? Could I use this to add custom settings to my program?

E.g. if I add a `foo: bar ` tag to the `args` in my `ProgramInterface.include_arguments` would it pass `foo=bar` to my runscript?

Reply:
Yes.

Reply:
That is create! Could we have the same for Arc & Dirac? We then wouldn't have to modify all the ` parse_arguments` in the `xyzrun.py` whenever someone wants to add a new option. E.g. rivet also had to be added to `nnlorun.py` in #41, even though NNNNNNLOJets doesn't use rivet.

Reply:
Yes, but the argument treatment for Arc and Dirac is a bit different so I'll do a separate PR so you guys can test it.